server/pulp/plugins/util/publish_step.py

```diff
@@ -577,9 +602,11 @@ def get_distributor_type(self):
 
     def publish(self):
         """
+        DEPRECATED
```

Comment:
Michael Hrivnak wrote:
> I'd love to remove it, but I'm sure there are plugins that are calling
> it. I think it'll be easier to deprecate a bunch of methods, then remove
> them later (like in 3.0).

We already have significantly broken backwards compatibility that I
don't think it's worth trying to preserve what little compatibility is
left. I think removing it is better since all the plugins have to be
touched as is anyway.

-- 
Randy Barlow
irc:   bowlofeggs
phone: +1-919-890-8150